### PR TITLE
Add mute button to toolbar

### PR DESCRIPTION
**Pull request by IntelOrca**
Supersedes #5389

### CODE REVIEW

File: src/openrct2/windows/top_toolbar.c [hunk 1]
```diff
@@ -781,6 +790,11 @@ static void window_top_toolbar_invalidate(rct_window *w)
 	else
 		w->pressed_widgets &= ~(1 << WIDX_PAUSE);
 
+	if (!gGameSoundsOff)
+		window_top_toolbar_widgets[WIDX_MUTE].image = 0x20000000 | SPR_G2_TOOLBAR_MUTE;
```
Comment (Gymnasiast) [inline src/openrct2/windows/top_toolbar.c]: It would be better to use a constant, rather than the magic number `0x20000000`.

Comment (IntelOrca) [inline src/openrct2/windows/top_toolbar.c]: Do it here, do it everywhere. I don't want to go down that road for this PR.

**Comment by IntelOrca** [comment]
The toolbar is broken, the footpath, scenery and land tools do not open / switch properly.

**Comment by marijnvdwerf** [comment]
https://github.com/OpenRCT2/OpenRCT2/pull/5018/files#diff-f553f4354f98a3b7629815b25d95bb62L358

**Comment by pulkitraggarwal** [comment]
The game crashes when the you try to play on my end. 
Invalid entry in g2.dat requested, idx = 87. You may have to update your g2.dat.
Along with that, On the title page at the left bottom it mentions "Issue #1421" which was a branch that i was locally working on. 
@IntelOrca @Gymnasiast

**Comment by janisozaur** [comment]
@pulkitraggarwal well, just like the message says: update your g2.dat

**Pull request merged**